src/components/SEP/Proposals/AssignSEPMemberToProposal.tsx

```diff
@@ -31,25 +31,12 @@ const AssignSEPMemberToProposal: React.FC<AssignSEPMemberToProposalProps> = ({
     { title: 'Organisation', field: 'user.organisation' },
   ];
 
-  const members = SEPMembersData?.filter(sepMember => {
-    if (
-      !sepMember.roles.some(
-        role => role.shortCode.toUpperCase() === UserRole.SEP_REVIEWER
-      )
-    ) {
-      return false;
-    }
-
-    if (
```

Comment:
This was added as part of https://github.com/UserOfficeProject/user-office-frontend/commit/6bff85449df32089208e32ae35b64f7e3a260905#diff-bbc1e3d4874a31cf7b9c8fb475bac7292571b32488ac76fedfb9e1c97f874a4d

Removing this may be a regression.

Reply:
I talked a bit with @bolmsten and I think we should keep SEP_Chair and SEP_Secretary as reviewers as well. They could do a review and you should be able to select them from that list of reviewers.

Reply:
I see, then it's fine by me.